src/recipe/Baked.Recipe.Service.Application/Runtime/RuntimeExtensions.cs

```diff
@@ -1,19 +1,48 @@
 ﻿using Baked.Architecture;
-using Baked.Configuration;
+using Baked.Runtime;
 using Baked.Testing;
 using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using System.Reflection;
 
 namespace Baked;
 
-public static class ConfigurationExtensions
+public static class RuntimeExtensions
 {
-    public static void AddConfiguration(this List<ILayer> layers) =>
-        layers.Add(new ConfigurationLayer());
+    public static void AddRuntime(this List<ILayer> layers) =>
+        layers.Add(new RuntimeLayer());
 
-    public static void ConfigureConfigurationBuilder(this LayerConfigurator configurator, Action<IConfigurationBuilder> configuration) =>
+    public static IServiceCollection GetServiceCollection(this ApplicationContext context) =>
+        context.Get<IServiceCollection>();
+
+    public static IServiceProvider GetServiceProvider(this ApplicationContext context) =>
+        context.Get<IServiceProvider>();
+
+    public static IServiceScope GetServiceScope(this ApplicationContext context) =>
+        context.Get<IServiceScope>();
+
+    public static void ConfigureLoggingBuilder(this LayerConfigurator configurator, Action<ILoggingBuilder> configuration) =>
+       configurator.Configure(configuration);
+
+    public static void ConfigureServiceCollection(this LayerConfigurator configurator, Action<IServiceCollection> configuration) =>
         configurator.Configure(configuration);
 
+    public static void ConfigureServiceProvider(this LayerConfigurator configurator, Action<IServiceProvider> configuration) =>
+        configurator.Configure(configuration);
+
+    public static void AddFromAssembly(this IServiceCollection services, Assembly assembly)
+    {
+        var serviceAdderType = assembly.GetExportedTypes().SingleOrDefault(t => t.IsAssignableTo(typeof(IServiceAdder))) ?? throw new("`IServiceAdder` implementation not found");
+        var serviceAdder = (IServiceAdder?)Activator.CreateInstance(serviceAdderType) ?? throw new($"Cannot create instance of {serviceAdderType}");
+
+        serviceAdder.AddServices(services);
+    }
```

Comment:
this one seems to be in between configure target helpers, lets keep the order as 1. add layer, 2. context getters, 3. configuration targets, 4. other helpers, 5. mockers and stubbers